x/leveragelp/spec/incentivized_liquidation_system_spec.md

```diff
@@ -1,39 +1,68 @@
-
-# Specification: Incentivized Liquidation System
+# Specification: Incentivized Liquidation System v2
 
 ## Overview
-The Incentivized Liquidation System ensures the stability and health of the leverage market by allowing users and bots to monitor and liquidate undercollateralized positions and postions that met a certain price(stop loss). Liquidators are rewarded for their efforts, creating an incentivized mechanism to maintain market health.
+
+The Incentivized Liquidation System is designed to maintain the stability and health of the leverage market by allowing users and bots to monitor and liquidate undercollateralized positions or those that have met a specific price condition (stop loss). Liquidators are rewarded for their contributions, creating an incentivized mechanism to ensure market stability and timely intervention.
 
 ## Scalability Advantage
-Unlike a system that requires traversing and evaluating all positions in each block—which is not scalable—the incentivized liquidation approach delegates the monitoring and liquidation tasks to users and bots. This significantly reduces the computational load on the blockchain, making the system scalable and capable of handling a large number of positions efficiently.
+
+Unlike systems that require evaluating all positions at each block—a method that is computationally intensive and not scalable—the incentivized liquidation system delegates the tasks of monitoring and liquidation to users and bots. This delegation significantly reduces the computational burden on the blockchain, making the system scalable and capable of efficiently managing a large number of positions.
 
 ## Incentivized Liquidation by Users and Bots
 
-Users and bots can monitor positions and trigger liquidations when necessary. They are rewarded for their actions, ensuring continuous monitoring and prompt liquidation of undercollateralized positions and stop loss.
+Users and bots can actively monitor positions and trigger liquidations when certain criteria are met. This decentralized approach ensures continuous oversight and prompt liquidation of undercollateralized positions or those nearing the stop loss threshold.
 
 ### Pseudo Code
 
 ```plaintext
 function IncentivizedCheckAndLiquidate(ctx, positionId, liquidator):
     position = getPosition(ctx, positionId)
     health = calculateHealth(position)
-    
-    if health < LIQUIDATION_THRESHOLD || lp_price <= stop_loss_price::
+
+    if health < LIQUIDATION_THRESHOLD or lp_price <= stop_loss_price:
```

Comment:
should we change this to `lp_price <= stop_loss_price * 1.01` to match it with the spec below ?